File: Python/film.py
class Film:
    def __init__(self, idFilm, Judul, Genre, Durasi, TahunRilis):
        self.__idFilm = str(idFilm)
        self.__Judul = str(Judul)
        self.__Genre = str(Genre)
        self.__Durasi = int(Durasi)
        self.__TahunRilis = int(TahunRilis)
    
    # Getter for all atribute
    def getIdFilm(self) -> str:
        return self.__idFilm
    def getJudul(self) -> str:
        return self.__Judul
    def getGenre(self) -> str:
        return self.__Genre
    def getDurasi(self) -> int:
        return self.__Durasi
    def getTahunRilis(self) -> int:
        return self.__TahunRilis
  
    # Setter for all atribute
    def setIdFilm(self, idFilm:str) -> None:
        self.__idFilm = str(idFilm)
    def setJudul(self, Judul:str) -> None:
        self.__Judul = str(Judul)
    def setGenre(self, Genre:str) -> None:
        self.__Genre = str(Genre)
    def setDurasi(self, Durasi:int) -> None:
        self.__Durasi = int(Durasi)
    def setTahunRilis(self, TahunRilis:int) -> None:
        self.__TahunRilis = int(TahunRilis)
   

File: PHP/film.php
<?php

class Film
{
    private $idFilm;
    private $judul;
    private $genre;
    private $durasi;
    private $tahunRilis;
    private $gambar;

    // Constructor
    public function __construct($idFilm, $judul, $genre, $durasi, $tahunRilis, $gambar)
    {
        $this->idFilm = $idFilm;
        $this->judul = $judul;
        $this->genre = $genre;
        $this->durasi = $durasi;
        $this->tahunRilis = $tahunRilis;
        $this->gambar = $gambar;
    }

    // Getter dan Setter ID Film
    public function getIdFilm()
    {
        return $this->idFilm;
    }
    public function setIdFilm($idFilm)
    {
        $this->idFilm = $idFilm;
    }

    // Getter dan Setter Judul
    public function getJudul()
    {
        return $this->judul;
    }
    public function setJudul($judul)
    {
        $this->judul = $judul;
    }

    // Getter dan Setter Genre
    public function getGenre()
    {
        return $this->genre;
    }
    public function setGenre($genre)
    {
        $this->genre = $genre;
    }

    // Getter dan Setter Durasi
    public function getDurasi()
    {
        return $this->durasi;
    }
    public function setDurasi($durasi)
    {
        $this->durasi = $durasi;
    }

    // Getter dan Setter Tahun Rilis
    public function getTahunRilis()
    {
        return $this->tahunRilis;
    }
    public function setTahunRilis($tahunRilis)
    {
        $this->tahunRilis = $tahunRilis;
    }

    // Getter dan Setter Gambar
    public function getGambar()
    {
        return $this->gambar;
    }
    public function setGambar($gambar)
    {
        $this->gambar = $gambar;
    }
}

?>